runtime/src/main/java/org/corfudb/runtime/view/ObjectsView.java

```diff
@@ -48,12 +46,17 @@
     @Setter
     boolean transactionLogging = false;
 
-
     @Getter
     Map<ObjectID, Object> objectCache = new ConcurrentHashMap<>();
 
+    Timer txCommitDurationTimer;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651478)